### ISSUE TITLE
Help files redesign (including a 'reference')

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#1332]
In https://github.com/pure-data/pure-data/issues/1320 we were discussing a new design format that would include a quick reference subpatch, much like in use in other documentation templates for Pd out there.

Here's a first draft: 

<img width="1175" alt="Screen Shot 2021-05-29 at 17 26 35" src="https://user-images.githubusercontent.com/5360142/120083994-0fab0300-c0a3-11eb-9f63-ff382b837894.png">

There's also the plan to provide such help files and references as html files distributed with Pd and making them available on the internet as well. They would live in '8.topics' where we already have html for slop~ and could be renamed to 'reference'

**Comment by porres** [comment]
the proposal in https://github.com/pure-data/pure-data/issues/1326 relates to the possibility of providing an html reference

**Comment by umlaeute** [comment]
> They would live in '8.topics' where we already have html for slop~ and could be renamed to 'reference'

so we have both `doc/5.reference/` and `doc/8.reference`?

**Comment by porres** [comment]
> so we have both doc/5.reference/ and doc/8.reference?

yeah, that doesn't look good? Any suggestions? How about rename '5.reference' to '5.help.files' and '8.topics' to '8.html.reference'?
